code/modules/mob/living/carbon/human/life.dm

```diff
@@ -973,6 +973,31 @@
 			holder.icon_state = "[pulse()]"
 		hud_list[HEALTH_HUD] = holder
 
+	if (BITTEST(hud_updateflag, BLINK_HUD) && hud_list[BLINK_HUD])
+		var/image/holder = hud_list[BLINK_HUD]
+		var/blink_time_current = null
+		var/blink_time_max = null
+		var/mob/living/scp_173/current173
+
+		for(var/mob/living/scp_173/A in GLOB.scp173s) //Gets the blink timer for the victim(mob that can see 173)
+			var/list/next_blinks = A?.next_blinks
+			var/list/next_blinks_join_time = A?.next_blinks_join_time
```

Comment:
tbh i think i did an oopsie suggesting `?.` here as we're pretty sure `A` exists
```suggestion
			var/list/next_blinks = A.next_blinks
			var/list/next_blinks_join_time = A.next_blinks_join_time
```

Reply:
my bad